src/styles/abstracts/_mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin letter-size-mobile($width, $height) {
```

Comment:
1. I'm thinking about the name… is anything `mobile` about these styles? there are no media queries here
2. maybe this will be against something you've read but my opinion is that if this code is not expected to be reused (this is the case here I suppose) it should be kept closer to the component where it's used in its styles file, it would better support the idea of a component as a self-contained entity, what do you think? would you like to convince me?
3. not sure about this a little bit magical values `0.8` or `0.91`, seems to be arbitrary, why is it needed? perhaps `overflow: hidden` may help with styling this? but maybe there is a good reason I can't identify so please advise